Makefile

```diff
@@ -82,6 +82,12 @@ install_examples: install
 build_examples:
 	go build -o ./examples/hotrod/hotrod-demo ./examples/hotrod/main.go
 
+build-all-in-one-linux:
+	cd jaeger-ui && npm install && npm run build
+	rm -rf jaeger-ui-build && mkdir jaeger-ui-build
+	cp -r jaeger-ui/build jaeger-ui-build/
```

Comment:
nit: move the above 3 lines to `build-ui` target and make this one depend on it (or even not depend)